samza-api/src/main/java/org/apache/samza/table/AsyncReadWriteTable.java

```diff
@@ -39,7 +39,7 @@
    * @return completableFuture for the requested value
    * @throws NullPointerException if the specified {@code key} is {@code null}.
    */
-  CompletableFuture<V> getAsync(K key);
+  CompletionStage<V> getAsync(K key);
```

Comment:
why make this change? It may break LI or the external users as well.

Reply:
+1.

This would affect for not just high-level API users, but also for any samza user who is using the `Table` API through `TaskContext` through low-level API.  

Would be better not to break backwards-compatibility here just for unification.

Reply:
As we don't have a lot of customer yet, let's make table API consistent with the rest of the low/high level API.